### ISSUE TITLE
List players on all proxied servers in tab-list

### ISSUE THREAD

**Issue by rickiewars** [minekube/gate#422]
Is it possible to list all players online on each server when the player views a tablist?

e.g.
| server 1| server 2 | server 3 |
|--------|--------|--------|
| player 1| player 3 | player 6 |
| player 2 | player 4 |  |
| | player 5 | |
| | | | 

I've looked into the source-code and documentation but couldn't figure out how I could do that.

**Comment by robinbraemer** [comment]
Yes, you can modify the tablist for every player via https://github.com/minekube/gate/blob/133de6f30cfec5b52036fd0f2cfc5776a85463d7/pkg/edition/java/proxy/player.go#L93 and add entries.

For further questions join the discord :D
